main.tf

```diff
@@ -33,11 +33,10 @@ resource "aws_ecs_task_definition" "agent-controller" {
     image = var.agent_controller_image
     essential = true
     portMappings = [{
-      name          = "aembit_agent_controller_http"
-      containerPort = 80
-      hostPort      = 80
+      name          = "aembit_agent_controller_https"
+      containerPort = 443

```

Comment:
Can you please check with Fatih + David on which ports it will be running in the container?

Bt default (on VM) Agent Controller uses ports
5000
5443

Just want to check that this will be a used port for TCP.

Reply:
@quantifex - can you confirm that the AC container uses ports 80/443, due to that .NET env var being set?

Reply:
Confirmed that this is _not_ being set be default in the container.